charts/prometheus-pingmesh-exporter/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "v1.0.0"
+appVersion: "v1.1.0"
```

Comment:
just raising it here will not work.
the deployment would be needed to be adjusted like in: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-conntrack-stats-exporter/templates/daemonset.yaml#L35

Reply:
OK. Pushed Docker image to DockerHub. 
https://hub.docker.com/repository/docker/dongjiang1989/pingmesh-agent/tags?page=1&ordering=last_updated&name=v1.1.0